joomla/joomla-cms
# [4.0] Installation: Fix arrow to be visible in dropdown

Pull Request for Issue #28050 .

### Summary of Changes
Arrow in dropdown is not visible during installation.
Replace light arrow with dark arrow svg files.

### Testing Instructions
Apply PR.
Delete configuration.php.
Do new install.
See visible arrow in dropdown.
